### DISCUSSION TITLE
Is there a way to open files directly from USB device?

### DISCUSSION THREAD

**Discussion by arrogantworm** [winapps-org/winapps#147] [Support] [answered]
It says /run/media/$USER/$DEVICE_NAME is not a correct path. I managed to pass the usb device as 'local share', but I can't open files from usb directly this way. I have to open excel - open file - select file.
Is there a way to pass usb to docker, so I can open files natively?

**Comment by MrTumnis** [comment, marked as answer]
This is explained in the [duckur/windows](https://github.com/dockur/windows) repo.
